openpype/hosts/hiero/api/menu.py

```diff
@@ -154,7 +160,7 @@ def add_scripts_menu():
         return
 
     # load configuration of custom menu
-    project_settings = get_project_settings(os.getenv("AVALON_PROJECT"))
+    project_settings = get_project_settings(get_current_project_name())
```

Comment:
There's also a `get_current_project_settings` which could be used here?

Reply:
Function `get_current_project_settings` probably should not exist. It can't access `openpype.pipeline` so it has to know where to look for project name on it's own which is not ideal.